src/calculatoradvance/Action.kt

```diff
@@ -0,0 +1,59 @@
+package calculatoradvance
+
+class Plus_operation() : Operation {
```

Comment:
general pattern for class naming 
PlusOperation. 

RENAME IT.

read this https://kotlinlang.org/docs/reference/coding-conventions.html